tests/app/celery/test_process_ses_receipts_tasks.py

```diff
@@ -575,3 +613,35 @@ def test_process_ses_results_personalisation(notify_db_session, sample_template,
     notify_db_session.session.refresh(notification)
     assert notification.status == NOTIFICATION_DELIVERED
     assert notification.personalisation == {'name': '<redacted>'}
+
+
+def ses_notification_complaint_callback(reference):
```

Comment:
This seems like something that should be at the top of the file, not the bottom.